lib/src/package_utils.dart

```diff
@@ -20,7 +20,11 @@ Map _getPubspec(String directoryName) {
   return loadYaml(contents);
 }
 
-String getPackageDescription(String directoryName) {
+String getPackageDescription(bool isSdk, String readmeLoc, String directoryName) {
+  if (isSdk && readmeLoc !=  null) {
+    var readmeFile = new File(readmeLoc);
+    if (readmeFile.existsSync()) return readmeFile.readAsStringSync();
```

Comment:
if it doesn't, what happens? when do we tell the user the filename that they provided is invalid?